src/main/java/chessmaster/commands/ShowMovesCommand.java

```diff
@@ -0,0 +1,29 @@
+package chessmaster.commands;
+
+import chessmaster.exceptions.ChessMasterException;
+import chessmaster.game.ChessBoard;
+import chessmaster.game.Coordinate;
+import chessmaster.pieces.ChessPiece;
+
+public class ShowMovesCommand extends Command{
+    public static final String SHOW_MOVE_COMMAND_STRING = "moves";
+    private static final String SHOW_MOVE_MESSAGE = "These are the moves available for the piece at %s.";
+    private String userInput;
+    private ChessPiece piece;
+
+    public ShowMovesCommand(String userInput){
+        this.userInput = userInput;
+    }
+
+    @Override
+    public CommandResult execute(ChessBoard board) throws ChessMasterException {
```

Comment:
- Execute `board.showAvailableMoves` here. So the board with available moves will show first before the return text
- Can have a line saying the type of piece the user is planning to move 
    - e.g. `You are moving Queen at a8. Here are the available moves: `
- Would be a good idea to tell the user the coordinates one can go to in text form
    - e.g. `a2, a3, b4, b5, b6...`
- You can pass in `String[]` into `CommandResult` for multi-line texts and each entry in the array will be printed on a new line